pkg/controller/worker/machines_test.go

```diff
@@ -549,11 +550,24 @@ var _ = Describe("Machines", func() {
 								Image:   machineImage,
 							},
 						},
-					}))
+					}
+
+					workerWithExpectedImages := w.DeepCopy()
+					workerWithExpectedImages.Status.ProviderStatus = &runtime.RawExtension{
+						Object: expectedImages,
+					}
+					ctx := context.TODO()
+					c.EXPECT().Get(ctx, gomock.Any(), gomock.AssignableToTypeOf(&extensionsv1alpha1.Worker{})).
```

Comment:
```suggestion
					c.EXPECT().Get(ctx, gomock.Any(), gomock.AssignableToTypeOf(&extensionsv1alpha1.Worker{})).Return(nil)
```

Reply:
You meant this right? - https://github.com/gardener/gardener-extension-provider-gcp/pull/183/commits/082dd6801beb95952bd04c5912f3d3d01497398e#diff-e61377d85357d3c316ed34aa40a57057R560